src/loaders/gtdb/gtdb_loader_helper.py

```diff
@@ -1,4 +1,5 @@
 import re
+from collections import defaultdict
```

Comment:
This seems like a general loader helper vs. gtdb specific, maybe it needs to be moved to genome_attributes?

Reply:
Except for parse_genome_id and maybe copy column

Reply:
👍 moved it to genome_attributes

Reply:
I think the gtdb specific functions should probably stay in /gtdb/, but good enough